src/oc/ftime.cpp

```diff
@@ -29,33 +29,35 @@ double nrn_time(void)
 	return ((double)time(&start_time));
 #endif
 }
-void hoc_startsw(void)
-{
-	Ret(nrn_time());
+
+
+
```

Comment:
nitpick : at multiple occasions we have add extra spaces; could be cleaned-up in clang-format PR.

Reply:
yeah, I have a clang-format PR on my list. Shall be done after major refactoring